include/deal.II/base/symmetric_tensor.h

```diff
@@ -2096,6 +2099,42 @@ SymmetricTensor<rank,dim,Number>::serialize(Archive &ar, const unsigned int)
 
 
 /**
+ * Add two symmetric tensors. If possible, you should use <tt>operator +=</tt>
+ * instead since this does not need the creation of a temporary.
+ *
+ * @relates SymmetricTensor
+ */
+template <int rank, int dim, typename Number, typename OtherNumber>
+inline
+SymmetricTensor<rank, dim, typename ProductType<Number, OtherNumber>::type>
+operator + (const SymmetricTensor<rank, dim, Number>      &left,
```

Comment:
Oh, and got even worst when one accumulated complex constructions of `AdditionType`s, `SumType`s and `ProductTypes`...